app/notification_push.go

```diff
@@ -136,6 +137,7 @@ func (a *App) sendPushNotification(notification *PostNotification, user *model.U
 	cfg := a.Config()
 	channel := notification.Channel
 	post := notification.Post
+	post.Message = utils.StripMarkdown(post.Message)
```

Comment:
We should do the processing in the asynchronous queue for push notification. Let's move this inside the `*App.sendPushNotificationSync` method instead.